src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -84,26 +80,29 @@ async def set_output_volts(self, voltage):
             :py:meth:`set_tpz_io_settings()<Tpz.set_tpz_io_settings>`
             method between the three values 75 V, 100 V and 150 V.
         """
+        if self.voltage_limit is None:
+            raise ValueError("Voltage limit is not set")
+
         if voltage < 0 or voltage > self.voltage_limit:
             raise ValueError(
                 "Voltage must be in range [0;{}]".format(self.voltage_limit)
             )
         volt = int(voltage * 32767 / self.voltage_limit)
-        payload = st.pack("<HH", 1, volt)
+        payload = st.pack("<HH", _CHANNEL, volt)
         await self.send(Message(MGMSG.PZ_SET_OUTPUTVOLTS, data=payload))
 
-    async def get_output_volts(self):
+    async def get_output_volts(self) -> float:
         """Get the output voltage applied to the piezo actuator.
 
         :return: The output voltage.
         :rtype: float
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_OUTPUTVOLTS, [MGMSG.PZ_GET_OUTPUTVOLTS], 1
+            MGMSG.PZ_REQ_OUTPUTVOLTS, [MGMSG.PZ_GET_OUTPUTVOLTS], _CHANNEL
```

Comment:
```suggestion
            MGMSG.PZ_REQ_OUTPUTVOLTS, [MGMSG.PZ_GET_OUTPUTVOLTS], Tpz._CHANNEL
```
Use class constants.